service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -43,7 +43,11 @@
               "iam.serviceAccounts.list",
               "lifesciences.workflows.run",
               "lifesciences.operations.cancel",
-              "serviceusage.services.use")
+              "serviceusage.services.use",
```

Comment:
Could you alphabetize these please?

Reply:
It would be nice to have a comment on each permission or group of related ones explaining why we this user category needs it.

Reply:
sorted!
I'm not sure how much comments would add here. All the permissions in these list (minus the already-commented service account ones) are at the project level because they can't be assigned at the resource level, and I think commenting what each permission does in GCP seems like it would be more noise than signal